client/components/CompanyProfile/FounderGroup.tsx

```diff
@@ -45,55 +48,56 @@ const BylineText = styled.p`
   line-height: 150%;
 `;
 
-export default () => (
-  <Wrapper>
-    <Founder>
-      <FounderIcon>
-        <img src="/static/Founder_Icon.png" />
-      </FounderIcon>
-      <div>
-        <p>
-          <a href="">Mac Miller</a>
-        </p>
-        <p>University of Pennsylvania</p>
-        <BylineText>Undergraduate</BylineText>
-        <BylineText>Male</BylineText>
-        <BylineText>
-          Caucasian, Asian, African, Latinx, Pacific Islander
-        </BylineText>
-      </div>
-    </Founder>
-    <Founder>
-      <FounderIcon>
-        <img src="/static/Founder_Icon.png" />
-      </FounderIcon>
-      <div>
-        <p>
-          <a href="">Mac Miller</a>
-        </p>
-        <p>University of Pennsylvania</p>
-        <BylineText>Undergraduate</BylineText>
-        <BylineText>Male</BylineText>
-        <BylineText>
-          Caucasian, Asian, African, Latinx, Pacific Islander
-        </BylineText>
-      </div>
-    </Founder>
-    <Founder>
-      <FounderIcon>
-        <img src="/static/Founder_Icon.png" />
-      </FounderIcon>
-      <div>
-        <p>
-          <a href="">Mac Miller</a>
-        </p>
-        <p>University of Pennsylvania</p>
-        <BylineText>Undergraduate</BylineText>
-        <BylineText>Male</BylineText>
-        <BylineText>
-          Caucasian, Asian, African, Latinx, Pacific Islander
-        </BylineText>
-      </div>
-    </Founder>
-  </Wrapper>
-);
+export interface FounderGroupProps {
+  company: Company;
+}
+
+export default class FounderGroup extends React.Component<FounderGroupProps> {
+  render() {
+    const founders = getAnswerValueFromRef(
+      this.props.company.typeform_data,
+      refsMap.founders
+    ).split(',');
+    const linkedins = getAnswerValueFromRef(
+      this.props.company.typeform_data,
+      refsMap.linkedins
+    ).split(',');
+    const schools = getAnswerValueFromRef(
+      this.props.company.typeform_data,
+      refsMap.schools
+    );
+    const gender_breakdown = getAnswerValueFromRef(
+      this.props.company.typeform_data,
+      refsMap.gender_breakdown
+    );
+    const race_or_ethnicity = getAnswerValueFromRef(
+      this.props.company.typeform_data,
+      refsMap.race_or_ethnicity
+    );
+
+    return (
+      <Wrapper>
+        {founders.map((founder) => (
+          <Founder>
+            <FounderIcon>
+              <img src="/static/Founder_Icon.png" />
```

Comment:
Maybe gravatar this?

Reply:
For founders? Founders generally don't upload an image

Reply:
Wait so why do we even have the icons here lol

Reply:
UX-wise, it makes it more obvious that its a founder. Better information hierarchy. It was just gonna be the same icon of a person regardless of what the actual person looks like